src/assets.rs

```diff
@@ -226,33 +222,71 @@ impl<'a> DerefMut for AssetEditor<'a> {
 
 /// Assets stored in an assets store.
 pub struct Assets {
+    /// A key-value store for assets.
     pub store: AssetsStore,
+
+    /// A set of asset codes loaded from verified asset libraries.
+    pub verified_assets: HashSet<AssetCode>,
 }
 
 impl Assets {
     #![allow(dead_code)]
 
-    /// Create assets.
+    /// Create unverified assets.
+    ///
+    /// To load verified assets afterwards, use `verify_asset`.
```

Comment:
`verify_assets`

Reply:
I'm not sure "Create unverified assets" is right, here. We're not actually creating any assets. What we're doing is creating (or loading!) a store of assets. In the detailed explanation, you can add a caveat that none of the loaded assets will be verified until `verify_assets` is called separately.